test/onnx/test_pytorch_onnx_onnxruntime_cuda.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import onnxruntime  # noqa
+import torch
+
+from test_pytorch_common import skipIfUnsupportedMinOpsetVersion
+from test_pytorch_common import skipIfNoCuda
+
+from test_pytorch_onnx_onnxruntime import TestONNXRuntime
+
+class TestONNXRuntime_cuda(unittest.TestCase):
```

Comment:
Thanks for setting this up. Are you going to add a CI job for these test?

Reply:
I see that this test is already added to CI job accidentally, i'll update to disable it for now.